soda-core/tests/features/test_dataset_filter.py

```diff
@@ -81,9 +81,11 @@ def test_dataset_filter(data_source_test_helper: DataSourceTestHelper):
               - missing:
     """
 
+    time_machine.set_now(t1)
+
     # On the first time partition t1 (16th) the filter should pass
     contract_verification_result_t1: ContractVerificationResult = data_source_test_helper.assert_contract_pass(
-        test_table=test_table, contract_yaml_str=contract_yaml_str, provided_now_timestamp=t1
```

Comment:
Can we get rid of the provided_now_timestamp parameter completely since it's not needed any more? Or is it someething you would like to use for backfilling/freshness or anything like that?

Reply:
provided_now_timestamp is completely replaced with the line above `time_machine.set_now(t1)`  No need to keep the provided now timestamp for other use cases later.